app/controllers/spree/admin/users_controller.rb

```diff
@@ -72,6 +72,13 @@ def clear_api_key
         redirect_to spree.edit_admin_user_path(@user)
       end
 
+      def toggle_api_key_view
+        if @user.toggle_api_key_view!
+          flash[:success] = t('spree.api.show_api_key_view_toggled')
+        end
+        redirect_to spree.edit_admin_user_path(@user)
+      end
```

Comment:
Can we use the standard `update` method instead? You can still create a custom flash message if this attribute changed.

Reply:
Yeah, of course! I just wanted to handle it the same way like the other api token actions. Should these methods be refactored into the update action as well?

Reply:
> Should these methods be refactored into the update action as well?

Yes. That would be great and reduce further confusion.